src/reducers/simpleReducer.js

```diff
@@ -18,11 +19,21 @@ const singleTrackReducer = (isLeftReducer) =>
             album: action.album,
             picture: action.picture,
             artist: action.artist,
-            trackLength: action.trackLength
+            trackLength: action.trackLength,
+            audio: action.audio
+          }
+      case 'PLAY AUDIO':
+          return {
+            ...state,
+            isAudioPlaying: action.isAudioPlaying
+          }
+      case 'PAUSE AUDIO':
+          return {
+            ...state,
+            isAudioPlaying: !action.isAudioPlaying
```

Comment:
hmm jeżeli pałzujemy lub chcemy, żeby coś zagraało to możemy w reducerze założyć stałe wartości czyli zawsze true lub zawsze false, dlatego przekazywanie tego z akcji nie jeste potrzebne

Reply:
done